openedx/features/content_tagging/rest_api/v1/urls.py

```diff
@@ -6,10 +6,13 @@
 
 from django.urls.conf import path, include
 
+from openedx_tagging.core.tagging.rest_api.v1 import views as oel_tagging_views
+
 from . import views
 
 router = DefaultRouter()
 router.register("taxonomies", views.TaxonomyOrgView, basename="taxonomy")
+router.register("object_tags", oel_tagging_views.ObjectTagView, basename="object_tag")
```

Comment:
TODO before merge:

Need to merge/tag https://github.com/openedx/openedx-learning/pull/68 and bump the installed openedx-learning version before this PR can be merged.

Reply:
@pomegranited My branch is actually based off of Romulo's branch https://github.com/openedx/edx-platform/pull/32871/ , since he already had the `rest_api` file structure setup (hence why the changes in my PR are very small) and since I thought his PR might be merging before mine, I kept it that way. If we think we'd want to have them separated, I can close this PR and open a new one against upstream master?

Reply:
Oh sure @yusuf-musleh, I'm happy to merge it to @rpenido 's branch, that makes sense.
We still need https://github.com/openedx/openedx-learning/pull/68's to be merged and tagged first, though.